homework/style.css

```diff
@@ -1,3 +1,94 @@
 .alert-error {
-    color: red;
-}
\ No newline at end of file
+  color: red;
+  background-color: #f8d7da;
+}
+.header {
+  color: white;
+  background-color: #3f51b5;
+  padding: 8px 16px;
+  margin-bottom: 16px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+.h1 {
+  font-size: 16px;
+  padding-right: 2%;
+  font-weight: 500;
+}
+.selectDiv {
+  background-color: #3f51b5;
+}
+#root {
+  max-width: 650px;
+  margin: 0 auto;
+}
+/* td:first-of-type {
+  font-weight: bold;
+}
+p:first-of-type {
+  background-color: dodgerblue;
+} */
+.image {
+  width: 60px;
+  height: 60px;
+  order: -1;
+}
+
+.repo-div {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+.left-div,
+.right-div {
+  margin-bottom: 8px;
+  border-radius: 2px;
+  background-color: #fff;
+  box-shadow: 4px 3px 19px -5px rgba(0, 0, 0, 0.75);
+  padding: 2%;
+  flex: 1;
+}
+.right-div {
+  margin-left: 5px;
+}
+.left-div {
+  margin-right: 5px;
+}
```

Comment:
In lines 52-57 you are making the assumption that the divs will always be displayed side by side. This does not hold true on a small screen width.